autoload/vundle/scripts.vim

```diff
@@ -254,4 +254,33 @@ func! s:load_scripts(bang)
   return eval(readfile(f, 'b')[0])
 endf
 
+
+" ---------------------------------------------------------------------------
+" Get the dependencies of the given bundle.
+"
+" bundle -- The bundle to check. (must be installed)
+" return -- List of bundle names.
+" ---------------------------------------------------------------------------
+func! vundle#scripts#getdeps(bundle)
+  if !empty(glob(a:bundle['rtpath'] . '/addon-info.json'))
+    let true = 1
+    let false = 0
+    let null = ''
```

Comment:
@jdevera: AFAIK they are constants in JSON, so they need to be defined when vim parses it.